### ISSUE TITLE
Login is not working starting from version 3.22

### ISSUE THREAD

**Issue by paulbauriegel** [gradio-app/gradio#3520]
### Describe the bug

The Login screen when the auth parameter is set does not work starting fom version 3.22.0. It still works in version 3.21.0.

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Reproduction

Try the Login Demo: https://github.com/gradio-app/gradio/blob/main/demo/hello_login/run.py
with the newest gradio version (3.22.1) 

### Screenshot

<img width="1114" alt="grafik" src="https://user-images.githubusercontent.com/20282916/226201483-ea1edf3e-f7b5-4f84-b788-9f35ed8aef08.png">

### Logs

```shell
After loading the Site

Uncaught (in promise) TypeError: z.stylesheets is undefined
    b http://127.0.0.1:7860/assets/index.8eda5c81.js:8
    zo http://127.0.0.1:7860/assets/index.8eda5c81.js:8
index.8eda5c81.js:8:22065
```
After pressing login
```
Uncaught (in promise) TypeError: Window.fetch: http://127.0.0.1:7860undefined/login is not a valid URL.
    u http://127.0.0.1:7860/assets/Login.b4c56691.js:2
    zr http://127.0.0.1:7860/assets/index.8eda5c81.js:4
    zr http://127.0.0.1:7860/assets/index.8eda5c81.js:4
    c http://127.0.0.1:7860/assets/Empty.svelte_svelte_type_style_lang.1fe8f058.js:1
Login.b4c56691.js:2:2797
Source-Map-Fehler: Error: NetworkError when attempting to fetch resource.
Ressourcen-Adresse: http://127.0.0.1:7860/assets/Login.b4c56691.js
Source-Map-Adresse: Login.b4c56691.js.map
```
Console with `debug=True`
```
paul@T0009e1196 ihub-webdemo % python login_demo.py
Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.
```
```


### System Info

```shell
Gradio: 3.22.1
Mac OSX Ventura
Firefox
```

### Severity

annoying

**Comment by abidlabs** [comment]
Can reproduce this issue, thanks @paulbauriegel! We'll fix it soon

**Comment by abidlabs** [comment]
Closed via #3529
